lib/bolt_server/pe/pal.rb

```diff
@@ -64,17 +67,35 @@ def initialize(plan_executor_config, environment_name, hiera_config = nil, max_c
           # environment: _technically_ that environment always exists, but if the dir
           # isn't there it won't find the module and fail with "plan not found" rather
           # than "environment doesn't exist"
-          if environment.configuration.path_to_env
-            bolt_config = File.join(environment.configuration.path_to_env, 'bolt.yaml')
+          if path_to_env
+            # This array is ordered: the first entry found that has a modulepath setting
+            # is what will be chosen
+            ['bolt.yaml'].each do |yaml_file|
```

Comment:
This array contains only one element.

Reply:
I think this is an artifact of us trying to add `bolt.yml` at one point. I'd say we shouldn't copy it into this repo, and should probably clean it up in Orchestrator.

Reply:
yeah, this was an artifact of when we were going to support both `bolt.yaml` and `bolt.yml`

We ended up taking that out but leaving in the rest of the processing code for when we put it back in.